CONTRIBUTING.md

```diff
@@ -570,6 +570,58 @@ It's a "to do" item and should be replaced if possible. `Any` is used when
 it's not possible to accurately type an item using the current type system.
 It should be used sparingly.
 
+### "The `Any` trick"
+
+Consider the following signature of `re.Match[str].group`:
```

Comment:
```suggestion
Consider the following (simplified) signature of `re.Match[str].group`:
```

Reply:
Not sure if this is a good idea. I'll let others decide.

Reply:
I was thinking about it too. It's a partial class signature. And only for one relevant overload of a method.